doc/devbox_setup.md

```diff
@@ -0,0 +1,115 @@
+This may not be sufficient to run certain samples that require Visual Studio and other SDKs or frameworks installed on your machine.
+
+### Windows + .NET Framework
+
+- Install the latest .NET Core from https://dot.net
+- Install .NET Framework 4.7 Developer Pack: https://support.microsoft.com/en-us/help/3186612/the-net-framework-4-7-developer-pack-and-language-packs
+- Install .NET Framework 4.5.1 Developer Pack: https://www.microsoft.com/en-us/download/details.aspx?id=40772
+- As admin (one-time setup):
+    Enable Powershell script execution on your system. See http://go.microsoft.com/fwlink/?LinkID=135170 for more information.
+
+### Linux/OSX (tested on Ubuntu 16.04)
+
+_Note:_ Developing on Linux limits the build to .NET Core only. The build system will not attempt to build .NET Framework (4.x).
+
+- Install Powershell https://github.com/PowerShell/PowerShell/blob/master/docs/installation/linux.md#ubuntu-1604
+- Install the latest .NET Core SDK CLI https://www.microsoft.com/net/learn/get-started/linux/ubuntu16-04
+
+<a name="advanced"/>
+
+## Optional Setup required to test Xamarin, Windows IoT and .NET Micro Framework
+
+### Installing Visual Studio 2017 for Xamarin applications
+
+- Install [Visual Studio 2017][visual-studio]. You can use the **Visual Studio Community** Free download if you meet the licensing requirements.
+- During the installation of Visual Studio 2017, we found that selecting the following workloads and components helps in running and debugging the SDK.
+
+Workloads:
+![](./workloads.png)
+
+Components:
+![](./components.png)
+
+### Installing Windows IoT Core SDK
+
+Install the Microsoft IoT Windows Core Project Templates for Visual Studio 2017 from the Extension Marketplace: 
+	https://marketplace.visualstudio.com/items?itemName=MicrosoftIoT.WindowsIoTCoreProjectTemplatesforVS15
+
+### Installing .NET Micro Framework
+
+#### 1. Download and Install Visual Studio Community 2015 Update 3 (approx. size: 9GB)
+
+NetMF is not supported on VS2017. We recommend a separate virtual machine with VS2015.
+VS2015 Community is sufficient. Download from: https://www.visualstudio.com/vs/older-downloads/
+
+Under Programming Languages, Visual C++ make sure that `Common Tools for Visual C++ 2015` is selected.
+
+#### 2. Download and install .NET MicroFramework from https://netmf.github.io/
+
+1. SDK v 4.4: 
+  * MicroFrameworkSDK.msi  (SDK v4.4) - Complete Installation
+  * NetMFVS14.vsx
+		
+2. SDK 4.3 - The official site has been archived. Either:
+  * Download the Codeplex archive (3.4GB) then unpack to find the MSI:
+    * Under releases\13 unpack c4ad01a9-68f6-48cd-82e5-e6d154b0eb2f
+    * Rename c4ad01a9-68f6-48cd-82e5-e6d154b0eb2f to netmf-v4.3.2-SDK-QFE2-RTM.zip and unzip
+    * Install MicroFrameworkSDK.msi - Complete Installation
+  * Download from http://developer.wildernesslabs.co/Netduino/About/Downloads/
+  
+3. Install .NET Framework 3.5
+
+<a name="testprereq"/>
```

Comment:
This structure is confusing... Do I need to download all 3? Do I download all three from netmf.github.io?

Reply:
> This structure is confusing... 

I agree. Let me restructure this.

> Do I need to download all 3? 
Yes

> Do I download all three from netmf.github.io?
No... will fix the title.